mirdata/utils.py

```diff
@@ -216,7 +217,7 @@ def __get__(self, obj, cls):
 
 
 class LargeData(object):
-    def __init__(self, index_file, metadata_load_fn=None):
+    def __init__(self, index_file, metadata_load_fn=None, download_path=None):
```

Comment:
I guess download_path will be used only in the case of loaders with remote indexes?

Reply:
I'm a bit confused about what is that you're passing to the downloader in `remotes` below. It should be a dictionary with a `RemoteFileMetadata` object but you're passing a path instead? Or I am missing something?

Reply:
>>I guess download_path will be used only in the case of loaders with remote indexes?
yes, of course!! By default it is not used....

>>I'm a bit confused about what is that you're passing to the downloader in remotes below. It should be a dictionary with a RemoteFileMetadata object but you're passing a path instead? Or I am missing something?
It is a dictionary with a RemoteFileMetadata object or None

Reply:
gotcha, maybe we can call it `remote_index` or something like that then? using the "_path" I'd expect a `str`

Reply:
done!!!